packages/core/admin/admin/src/content-manager/hooks/tests/useDocument.test.tsx

```diff
@@ -46,7 +47,7 @@ describe('useDocument', () => {
       useDocument({
         collectionType: 'collection-types',
         model: mockData.contentManager.contentType,
-        id: '12345',
+        documentId: '12345',
       })
```

Comment:
completely unrelated question, would it be possible to infer if the collectionType is collection-types or single-types based on the model?